Sources/SpecialErrorPages/SpecialErrorPageHTMLTemplate.swift

```diff
@@ -1,38 +0,0 @@
-//
-//  SpecialErrorPageHTMLTemplate.swift
-//
-//  Copyright © 2024 DuckDuckGo. All rights reserved.
-//
-//  Licensed under the Apache License, Version 2.0 (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//  http://www.apache.org/licenses/LICENSE-2.0
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//  See the License for the specific language governing permissions and
-//  limitations under the License.
-//
-
-import Foundation
-import ContentScopeScripts
-import WebKit
-import Common
-
-public struct SpecialErrorPageHTMLTemplate {
```

Comment:
I saw this function has been moved to macOS. It is used by iOS as well? It would be good to leave it in BSK otherwise we will need to duplicate it